### PR TITLE
Add s390x arch support

### PR DESCRIPTION
**Pull request by Repana-Chowdappa**
This is continuing of https://github.com/bazelbuild/rules_rust/pull/2557 , added buildifier code changes for s390x arch also.

### CODE REVIEW

**Comment by Repana-Chowdappa** [comment]
@UebelAndre - now we have buildifier binaries available for s390x arch from buildtools 7.3.1 https://github.com/bazelbuild/buildtools/releases/download/v7.3.1/buildifier-linux-s390x . Can we please update the same on https://github.com/bazelbuild/rules_rust/blob/ef8ac18bb44f8b480b7f50b6c031504a94785f2e/crate_universe/private/vendor_utils.bzl#L6?

**Review by UebelAndre — APPROVED** [review]
Looks like you just need to run buildifier one more time and the change is good to go!

**Comment by Repana-Chowdappa** [comment]
@UebelAndre - I don't have access to retry or rerun buildifier on CI, please provide me the access to the same.

**Comment by UebelAndre** [comment]
> @UebelAndre - I don't have access to retry or rerun buildifier on CI, please provide me the access to the same.

You would just run buildifier on whatever machine you can and push the results to your branch.

**Comment by Repana-Chowdappa** [comment]
@UebelAndre  - Can we upgrade the buildifier to 7.3.1? where we have s390x binary is available.

**Comment by UebelAndre** [comment]
> @UebelAndre - Can we upgrade the buildifier to 7.3.1? where we have s390x binary is available.

Yeah, go ahead and bump the version 😄

**Comment by Repana-Chowdappa** [comment]
@UebelAndre - I have fixed the buildifier CI failures, Can you please merge this PR, we need this for a customer deployment.

**Comment by Repana-Chowdappa** [comment]
@UebelAndre - All CI checks passed. Can you please help us on merging this PR.

**Review by UebelAndre — APPROVED** [review]
Thanks, looks good to me!

File: crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel [hunk 1]
```diff
@@ -48,6 +48,9 @@ rust_library(
         "@rules_rust//rust/platform:i686-unknown-linux-gnu": [
             "errno",  # i686-unknown-linux-gnu
         ],
+        "@rules_rust//rust/platform:s390x-unknown-linux-gnu": [
```
Comment (UebelAndre) [inline crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel:51]: I noticed on https://github.com/bazelbuild/rules_rust/pull/2841 that this change goes away when I regenerate dependencies. How was this produced?

Comment (Repana-Chowdappa) [reply crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel:51]: @UebelAndre - any specific reason to remove this?

**Comment by Repana-Chowdappa** [comment]
@UebelAndre - when will be the next release for rules_rust? so that we will have rust available for s390x platform.

File: crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel (hunk 1 above)
Comment (UebelAndre) [reply crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel:51]: These files are generated and if there is a change made to them they should be made in a way that regenerates into the file. The removal feels more like a miss where something was not hooked up correctly and I was wondering how these came to be in the first place.

**Pull request merged**